beets/util/__init__.py

```diff
@@ -524,30 +531,31 @@ def move(path: bytes, dest: bytes, replace: bool = False):
         # Copy the file to a temporary destination.
         basename = os.path.basename(bytestring_path(dest))
         dirname = os.path.dirname(bytestring_path(dest))
-        tmp = tempfile.NamedTemporaryFile(
+        tmp = NamedTemporaryFile(
             suffix=syspath(b".beets", prefix=False),
             prefix=syspath(b"." + basename, prefix=False),
             dir=syspath(dirname),
             delete=False,
         )
         try:
-            with open(syspath(path), "rb") as f:
+            with open(syspath(path)) as f:
                 shutil.copyfileobj(f, tmp)
```

Comment:
```suggestion
            with open(syspath(path), "rb") as f:
                # mypy bug:
                # - https://github.com/python/mypy/issues/15031
                # - https://github.com/python/mypy/issues/14943
                # Fix not yet released:
                # - https://github.com/python/mypy/pull/14975
                shutil.copyfileobj(f, tmp)  # type: ignore[misc]
```
Why remove the "rb"? I had another issue with mypy at this place, see the suggestion.

Reply:
Whoa, well caught! I changed precisely because `mypy` check was failing. I also didn't realise that `NamedTemporaryFile` opens the file in bytes mode by default. Just checked my change would actually make this fail.

I wonder why this isn't tested?

Reply:
That's actually unsurprising: This is only in the error handler for a cross-filesystem move. To test this, we would need to set up an extra filesystem (maybe on a loop device? mabye a bind-mount would suffice?). It would in fact be great to have that, since I suppose it's a common use case (e.g. have the music on some external device, but new files to import in your Downloads folder).

Reply:
This makes sense!